src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -351,14 +351,5 @@ public void SetComparer_SetEqualsTests()
             Assert.True(comparerSet2.SetEquals(set));
         }
         #endregion
-
```

Comment:
These sortedset changes are unrelated to your change and in fact were made by @stephentoub. I'm not sure how you managed to rebase such that they are showing up as a diff. I guess they will merge away.